### DISCUSSION TITLE
run latest g-w on HERA for S2SW

### DISCUSSION THREAD

**Discussion by jiandewang** [NOAA-EMC/global-workflow#2609] [General]
I cloned g-w couple of days ago (hash # bb930050, Thu May 16 12:28:30) and set up C768 S2SW. fcst job failed in the very beginning with error:
  10005 7457:  EXPTOTAL is selected
  10006 7457:  But PDLIB is not
11887 6760: [h21c38:1452014:0:1452014]    ud_iface.c:763  Fatal: transport error: Endpoint timeout
11900 6720:  8 0x00000000006d0b30 fi_cq_read()  /p/pdsd/scratch/Uploads/IMPI/other/software/libfabric/linux/v1.9.0/include/rdma        /fi_eq.h:385

see log file on HERA at: 
/scratch1/NCEPDEV/climate/Jiande.Wang/working/scratch/global-workflow/HR3-20191203/COMROOT/2019120300/HR3-20191203/logs/2019120300

**Comment by JessicaMeixner-NOAA** [comment]
@jiandewang what options did you use to build? 

The build option of -w for GFS is no longer needed (it's now the default and if you want to build GEFS you use the -w)

**Comment by jiandewang** [comment]
> @jiandewang what options did you use to build?
> 
> The build option of -w for GFS is no longer needed (it's now the default and if you want to build GEFS you use the -w)

I had -w. Let me retry without that. Thanks

**Reply by JessicaMeixner-NOAA** [reply]
That's likely the issue.

**Comment by jiandewang** [comment]
now I have
   0: PASS: fcstRUN phase 2, n_atmsteps =               57 time is         0.585069
   4:  ncells=           5
   4:  nlives=          12
   4:  nthresh=   18.0000000000000
4608: Abort(470417410) on node 4608 (rank 4608 in comm 0): Fatal error in PMPI_Type_contiguous: Invalid count, error stack:
4608: PMPI_Type_contiguous(271): MPI_Type_contiguous(count=-2057106627, MPI_BYTE, new_type_p=0x7ffdc6cc0194) failed
4608: PMPI_Type_contiguous(238): Negative count, value is -2057106627
   0: slurmstepd: error: *** STEP 60473108.0 ON h1c16 CANCELLED AT 2024-05-20T15:07:45 ***
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
srun: error: h22c39: tasks 6240-6279: Killed

logfile location: /scratch1/NCEPDEV/climate/Jiande.Wang/working/scratch/global-workflow/HR3-20191203/COMROOT/2019120300/HR3-20191203/logs/2019120300

**Reply by JessicaMeixner-NOAA** [reply]
You need a model update to get past that error on hera... and we can't update the model until the wcoss2 library issues are resolved.

**Comment by jiandewang** [comment]
@JessicaMeixner-NOAA thanks for the tip

**Comment by aerorahul** [comment]
@jiandewang Are you still having issues running g-w on Hera for S2SW?  The CI tests suggests they are running as expected.

**Reply by jiandewang** [reply]
You can close it now
